### DISCUSSION TITLE
How is this different than a style?

### DISCUSSION THREAD

**Discussion by gael-vanderlee** [UliGesing/Darktable-Initial-Workflow-Module#19] [Q&A] [answered]
I'm new to darktable so this might be a stupid question. But how is what this module does different then using a style ? Doesn't it functionally do the same thing: apply a bunch of modules to an image? 
Thanks for the help.

**Comment by UliGesing** [comment, marked as answer]
Hi @gael-vanderlee, sorry for my late answer. And no, it is not a stupid question :-)

Using a style and using this script are just two different approaches. Using a style is a rather static method, it sets some predefined settings without caring about the current image. The use of this script is more dynamic, it calls some automatisms of various modules (like "magic hands") in the darkroom view and configures some default settings. The result of these automatisms depends on your current image.

Using a style is significantly more performant and is directly supported by Darktable. However, there are often manual steps that follow for many images. The script can take care of some of these manual steps.

Hope that helps.

**Reply by gael-vanderlee** [reply]
I see, that makes sense. Thanks!
